src/content/about/using-cache.md

```diff
@@ -3,143 +3,39 @@ title: Using the Cache
-
-```javascript
-// Set cache key for this request to "some-string".
-fetch(event.request, { cf: { cacheKey: 'some-string' } })
-```
-
-Normally, Cloudflare computes the cache key for a request based on the request's URL. Sometimes, though, you'd like different URLs to be treated as if they were the same for caching purposes. For example, say your web site content is hosted from both Amazon S3 and Google Cloud Storage - you have the same content in both places, and you use a Worker to randomly balance between the two. However, you don't want to end up caching two copies of your content. You could utilize custom cache keys to cache based on the original request URL rather than the subrequest URL:
-
-```javascript
-addEventListener('fetch', event => {
-  let url = new URL(event.request.url)
-  if (Math.random() < 0.5) {
-    url.hostname = 'example.s3.amazonaws.com'
-  } else {
-    url.hostname = 'example.storage.googleapis.com'
-  }
-
-  let request = new Request(url, event.request)
-  event.respondWith(
-    fetch(request, {
-      cf: { cacheKey: event.request.url },
-    }),
-  )
-})
-```
-
-Remember, Workers operating on behalf of different zones cannot affect each other's cache. You can only override cache keys when making requests within your own zone (in the above example `event.request.url` was the key stored), or requests to hosts that are not on Cloudflare. When making a request to another Cloudflare zone (e.g. belonging to a different Cloudflare customer), that zone fully controls how its own content is cached within Cloudflare; you cannot override it.
-
-### Overriding Browser Cache (Cache-Control headers)
-
-Browsers determine what to cache through the response headers sent from Cloudflare.
+### [Cache API](/reference/apis/cache)
 
-```javascript
-async function fetchAndApply(request) {
-  let response = await fetch(request)
+The Cache API creates a private cache key namespace for each zone on Cloudflare’s CDN. Cache objects are stored in the local datacenter handling the request and are not replicated to other datacenters.
 
-  // Make response headers mutable
-  response = new Response(response.body, response)
+When to use Cache API:
 
-  response.headers.set('Cache-Control', 'max-age=1500')
+- Attempts to read from cache without calling fetch. (i.e. send me the response to slow.com/resource if and only if it’s already a HIT on cache is cached using `caches.default.match(..)`.
+- Explicitly store a response in the cache using `caches.default.put(..)` and explicitly delete `caches.default.delete(..)`. For example, the zone. https://fonts.googleapis.com/css is not gray clouded since it’s not on Cloudflare, so we could store https://fonts.googleapis.com/css as cache key on our zone’s namespace.
```

Comment:
Similar to the above, "When you must explicitly store a response ..."

I also think a better motivating example would be setting a Cache-Control on the response, in the event the origin cannot add one itself. Someone trying to cache `https://fonts.googleapis.com/css` would actually be better off using `fetch(..., { cf: { cacheTtl: nnn } })`. :)